lib/ueberauth/strategy/application.ex

```diff
@@ -4,14 +4,20 @@ defmodule Ueberauth.Strategy.Passwordless.Application do
   alias Ueberauth.Strategy.Passwordless
 
   def start(_type, opts) do
-    if Passwordless.config(:use_store), do: start_store(opts)
+    :use_store
+    |> Passwordless.config()
+    |> maybe_start_store(opts)
   end
 
-  defp start_store(opts) do
+  defp maybe_start_store(true, opts) do
```

Comment:
I think `maybe` is a very bad name for a function because you can not be sure what will happen.

Reply:
I do not agree with this. It is made obvious that the `Store` should be started according to some configuration.
Also the word `maybe` is used a lot in function naming, in Elixir too: https://github.com/elixir-lang/elixir/search?q=maybe&unscoped_q=maybe

Reply:
Maybe prefix with `try`?